src/classes/RD2_EnablementDelegate_TEST.cls

```diff
@@ -505,6 +505,55 @@ public with sharing class RD2_EnablementDelegate_TEST {
         System.assertEquals(null, job, 'A Queueable job after error on Enhanced Recurring Donations should not be running');
     }
 
+    /**
+    * @description Verifies that when no legacy RD exists in the org, the enablement page should load the simple version 
+    */
+    @isTest
+    private static void shouldIndicateSimplePageConfigWhenNoLegacyRdExists() {
+        Test.startTest();
+        Boolean isSimpleVersion = RD2_EnablementDelegate_CTRL.isSimpleVersion;
+        Test.stopTest();
+
+        System.assert(isSimpleVersion, 'The enablement page should be in simple version when no legacy RD exists');
+    }
+
+    /***
+    * @description Verifies that when a legacy RD exists, the enablement page should load the complex version 
```

Comment:
`complex` -> `whole` or `standard` or `full` (preferred). Let's use the same descriptor of the page in test method names as well.